pylsp/__main__.py

```diff
@@ -7,10 +7,7 @@
 import sys
 import time
 
-try:
-    import ujson as json
-except Exception:
-    import json
```

Comment:
Please restore this and all other try/excepts. That's necessary for people that run this project in environments such as Pyodide.

Reply:
Done. I removed this one in this file since it is only used to load the logging config; thus not performance critical.